### PR TITLE
docs: fix broken tables in browser-support.md

### PR DESCRIPTION
**Pull request by leandrodiniz**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by jelbourn** [comment]
@leandrodiniz it's not clear from your PR/commit description what's actually broken here; it looks like the changes are mostly formatting. Additionally, this change seems to have broken the link rendering in the "Optional browser features to polyfill" table (you can see in the preview link posted above)

**Comment by leandrodiniz** [comment]
@jelbourn actually it's mostly formatting changes but that impacts the way the tables are displayed on GitHub. You can check the differences here:

**master:**
https://github.com/angular/angular/blob/master/aio/content/guide/browser-support.md

**my branch:**
https://github.com/angular/angular/blob/2e89f2ac0f78651653824efd8cf4f36a74d4c274/aio/content/guide/browser-support.md

and I also fixed the link rendering in the 'polyfills' table that was broken.

**Comment by jelbourn** [comment]
Ah, I see. Can you squash your changes into a single commit with a message that clarifies that this addresses rendering _on GitHub_. Something like

```
docs: format browser-support tables for GitHub markdown

Format the HTML used in browser-support.md such that the tables
render correctly under GitHub markdown.
```

**Comment by leandrodiniz** [comment]
Done. I squashed my changes into a single commit with a clearer message.

File: aio/content/guide/browser-support.md [hunk 1]
```diff
@@ -3,89 +3,41 @@
...
-    <td>
-      IE
-    </td>
-    <td>
-      <div>11</div>
-    </td>
+    <td>IE</td>
+    <td>11</td>
   </tr>
  <tr>
```
Comment (gkalpak) [inline aio/content/guide/browser-support.md:26]: While you're at it, you could fix the indentation here too 🙏 

```suggestion
  <tr>
```

File: aio/content/guide/browser-support.md [hunk 2]
```diff
@@ -157,220 +109,133 @@ If the polyfill you want is not already in `polyfills.ts` file, add the `import`
...
     <td>
       Chrome, Firefox, Edge, <br>
       Safari, Android, IE 11
     </td>
-
     <td>
-
-      [ES2015](guide/browser-support#core-es6)
-
+    <a href="guide/browser-support#core-es6" class="code-anchor">ES2015</a>
```
Comment (gkalpak) [inline aio/content/guide/browser-support.md:122]:
```suggestion
      <a href="guide/browser-support#core-es6">ES2015</a>
```

File: aio/content/guide/browser-support.md [hunk 3]
```diff
@@ -157,220 +109,133 @@ If the polyfill you want is not already in `polyfills.ts` file, add the `import`
...
+    <th>Polyfill</th>
+    <th style="width: 50%">Browsers (Desktop & Mobile)</th>
   </tr>
-
   <tr style="vertical-align: top">
-
     <td>
-
-      [AnimationBuilder](api/animations/AnimationBuilder).
+      <a href="api/animations/AnimationBuilder" class="code-anchor">AnimationBuilder</a>
```
Comment (gkalpak) [inline aio/content/guide/browser-support.md:139]:
```suggestion
      <a href="api/animations/AnimationBuilder">AnimationBuilder</a>
```

File: aio/content/guide/browser-support.md [hunk 4]
```diff
@@ -157,220 +109,133 @@ If the polyfill you want is not already in `polyfills.ts` file, add the `import`
...
       (Standard animation support does not require polyfills.)
-
     </td>
-
     <td>
-
-      [Web Animations](guide/browser-support#web-animations)
-
+      <a href="guide/browser-support#web-animations" class="code-anchor">Web Animations</a>
```
Comment (gkalpak) [inline aio/content/guide/browser-support.md:143]:
```suggestion
      <a href="guide/browser-support#web-animations">Web Animations</a>
```

File: aio/content/guide/browser-support.md [hunk 5]
```diff
@@ -157,220 +109,133 @@ If the polyfill you want is not already in `polyfills.ts` file, add the `import`
...
     </td>
-
   </tr>
 
   <tr style="vertical-align: top">
-
     <td>
-
-       [NgClass](api/common/NgClass) on SVG elements
+      <a href="api/common/NgClass" class="code-anchor">NgClass</a> on SVG elements
```
Comment (gkalpak) [inline aio/content/guide/browser-support.md:154]:
```suggestion
      <a href="api/common/NgClass">NgClass</a> on SVG elements
```

File: aio/content/guide/browser-support.md [hunk 6]
```diff
@@ -157,220 +109,133 @@ If the polyfill you want is not already in `polyfills.ts` file, add the `import`
...
     </td>
-
     <td>
-
-      [classList](guide/browser-support#classlist)
-
+      <a href="guide/browser-support#classlist" class="code-anchor">classList</a>
```
Comment (gkalpak) [inline aio/content/guide/browser-support.md:157]:
```suggestion
      <a href="guide/browser-support#classlist">classList</a>
```

File: aio/content/guide/browser-support.md [hunk 7]
```diff
@@ -157,220 +109,133 @@ If the polyfill you want is not already in `polyfills.ts` file, add the `import`
...
-
   </tr>
 
   <tr style="vertical-align: top">
-
     <td>
-
-      [Router](guide/router) when using
-      [hash-based routing](guide/router#location-strategy)
+      <a href="guide/router" class="code-anchor">Router</a> when using <a href="guide/router#location-strategy" class="code-anchor">hash-based routing</a>
```
Comment (gkalpak) [inline aio/content/guide/browser-support.md:166]:
```suggestion
      <a href="guide/router">Router</a> when using <a href="guide/router#location-strategy">hash-based routing</a>
```

File: aio/content/guide/browser-support.md [hunk 8]
```diff
@@ -157,220 +109,133 @@ If the polyfill you want is not already in `polyfills.ts` file, add the `import`
...
     </td>
-
     <td>
-
-      [ES7/array](guide/browser-support#core-es7-array)
-
+      <a href="guide/browser-support#core-es7-array" class="code-anchor">ES7/array</a>
```
Comment (gkalpak) [inline aio/content/guide/browser-support.md:169]:
```suggestion
      <a href="guide/browser-support#core-es7-array">ES7/array</a>
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx, @leandrodiniz! 

OOC, why were you trying to view the file on GitHub? (Just making sure there are no links we need to update, because this file is not intended to be viewed in the source code directly.)

**Review by gkalpak — APPROVED** [review]
LGTM :+1: Thx, @leandrodiniz!
(FYI, I have rebased the PR on master to resolve a conflict and allow CI to run.)

OOC, why were you trying to view the file on GitHub? (Just making sure there are no links we need to update, because this file is not intended to be viewed in the source code directly.)

**Review by jelbourn — APPROVED** [review]
LGTM

Reviewed-for: docs-packaging-and-releasing

**Pull request closed without merging**